specification/compute/resource-manager/readme.md

```diff
@@ -206,6 +206,16 @@ directive:
       - ACS service is deprecated so a PATCH endpoint won't be implemented
 
 ```
+
+### Tag: package-2019-12-01
```

Comment:
Since default tag is still package-2019-07, 2019-12-01 version of gallery will not be included in official SDK.  Is it intended?